src/inspector/node_string.h

```diff
@@ -67,8 +78,42 @@ void builderAppendQuotedString(StringBuilder& builder, const String&);
 std::unique_ptr<Value> parseJSON(const String&);
 std::unique_ptr<Value> parseJSON(v8_inspector::StringView view);
 
+std::unique_ptr<Value> parseMessage(const std::string& message, bool binary);
+ProtocolMessage jsonToMessage(String message);
+ProtocolMessage binaryToMessage(std::vector<uint8_t> message);
+String fromUTF8(const uint8_t* data, size_t length);
+
 extern size_t kNotFound;
 }  // namespace StringUtil
+
+// A read-only sequence of uninterpreted bytes with reference-counted storage.
+// Though the templates for generating the protocol bindings reference
+// this type, js_protocol.pdl doesn't have a field of type 'binary', so
+// therefore it's unnecessary to provide an implementation here.
+class Binary {
+ public:
+  const uint8_t* data() const {
+    DCHECK(false);
```

Comment:
Does it make sense to `UNREACHABLE();`? You can drop the `return` statements in that case.

Reply:
Ah, perfect, that's what it is in v8/chromium, I was not sure if node prefers that as well.

Reply:
Correction, it was `UNIMPLEMENTED` in v8/chromium. Anyhow, using `UNREACHABLE` now